poc_iot_verifier/src/main.rs

```diff
@@ -69,10 +69,16 @@ impl Server {
         let loader = loader::Loader::from_settings(settings).await?;
         let runner = runner::Runner::from_settings(settings).await?;
         let purger = purger::Purger::from_settings(settings).await?;
+
+        // TODO: Construct density_scaler settings from verifier settings?
```

Comment:
I don't understand this todo quite yet..  afaict the density_scaler::Settings would be used in the Settings of the verifier